package.json

```diff
@@ -223,6 +223,8 @@
 		"test": "npm run lint && npm run test-unit",
 		"test-e2e": "wp-scripts test-e2e --config packages/e2e-tests/jest.config.js",
 		"test-e2e:debug": "wp-scripts --inspect-brk test-e2e --config packages/e2e-tests/jest.config.js --puppeteer-devtools",
+		"test-e2e:open-browser": "cross-env PUPPETEER_HEADLESS=false PUPPETEER_SLOWMO=80 wp-scripts test-e2e --config packages/e2e-tests/jest.config.js",
+		"test-e2e:devtools": "cross-env PUPPETEER_HEADLESS=false PUPPETEER_SLOWMO=80 PUPPETEER_DEVTOOLS=true wp-scripts test-e2e --config packages/e2e-tests/jest.config.js",
```

Comment:
You can also enable DevTools with a keyboard shortcut while running the tests in the browser so I don't think it deserves it's own script here.